packages/cozy-client/src/models/permission.js

```diff
@@ -46,7 +46,7 @@ export function isReadOnly(perm, options = {}) {
  * @param {CozyClient} client -
  * @returns {PermissionItem[]} list of permissions
  */
-async function fetchOwn(client) {
+export async function fetchOwn(client) {
```

Comment:
Could you remove the @private mention in the docstring ?
Also, could you replace Get by Fetch in the docstring ?